### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Benawi**
## Features Added:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: JS/Index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const MENU_NODES = [MOBILE_DROP_DOWN_BTN, MOBILE_EXPANDED_MENU];
+
+// Event listeners to open and close the mobile menu dropdown
+
+// for (let i = 0; i < MENU_NODES.length; i += 1) {
+//   MENU_NODES[i].addEventListener('click', () => {
+//     document.getElementById('mobile-expanded-menu').classList.toggle('disabled');
+//     document.querySelector('html').classList.toggle('no-overflow-y');
+//   });
+// }
```
Comment (CollinsTatang) [inline JS/Index.js]: - [ ] Please remove all [Commented Code](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) from your codebase to make it clean and easy to maintain

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Benawi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project matches the given figma design :heavy_check_mark: 

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
### CHANGES ♻️

- [Commented Code](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) removed from codebase .

**Review by osamaashraf6 — APPROVED** [review]
Hi @Benawi  😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF

## Highlights 🎉🎉
- [x] No linter errors present
- [x] The PR has a proper title and a descriptive body 👍
- [x] Nice design 💯

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**